frontend/apps/filemanager/filemanager.lua

```diff
@@ -253,19 +253,19 @@ function FileManager:setupLayout()
             {}, -- separator
         }
 
+        local book_props
         if is_file then
-            self.book_props = nil -- in 'self' to provide access to it in CoverBrowser
```

Comment:
(I guess this was obsolete, as there's no trace of `book_props` in coverbrowser ?)